Source/Public/Add-VSTeam.ps1

```diff
@@ -8,7 +8,7 @@
 # https://bit.ly/Add-VSTeam
 
 function Add-VSTeam {
-   [CmdletBinding()]
+   [CmdletBinding(HelpUri='https://methodsandpractices.github.io/vsteam-docs/modules/vsteam/Add-VSTeam')]

```

Comment:
@DarqueWarrior I hope you don't kill me, but due to technical improvements I had to change the basic url from

`https://methodsandpractices.github.io/vsteam-docs/modules/vsteam/`
to
`https://methodsandpractices.github.io/vsteam-docs/docs/modules/vsteam/`

there is the "docs" path being added. Nothing will change in the future. not sure if I can get rid of it.

Reply:
No problem that is an easy change with regular expressions. To confirm I just need to add 'docs' between 'vsteam-docs/' and '/modules/". Is that correct?

Reply:
Done

Reply:
yes confirmed!

Reply:
When will a new version of the site be pushed with the path change? I just noticed the new paths are 404s right now. I would like this to be updated again before I merge the latest changes into master.

Reply:
I can push it today in 1 hour. Just tell me soon after if we keep one module or split in two. I prepared sub doc pages for vsteam plus, but it is basically just menu and placeholder intro pages.